README.md

```diff
@@ -1,8 +1,36 @@
-Before you start
-=================
+# www.openshift.io
 
-Backend API
------------
+
+This repository contain the landing page content for openshift.io, a
+free, end-to-end development environment in the cloud.
+
+## Community Feedback
+
+
+If you have feedback, suggestions and ideas we are to be found at the
+following places:
+
+irc at [\#openshiftio on Freenode](irc://freenode.org/#openshiftio) or [webchat](http://webchat.freenode.net/?channels=%23openshiftio)
+
+planner/issues at [openshift.io](https://openshift.io/openshiftio/openshiftio/plan)
+
+stackoverflow with tag [openshiftio](http://stackoverflow.com/questions/tagged/openshiftio)
+
+## Repositories
+
+The upstream for openshift.io is the Fabric8 project.
+Fabric8 is found in the following organizations:
+
+<https://github.com/fabric8io>
+<https://github.com/fabric8-ui>
+<https://github.com/fabric8-analytics>
+<https://github.com/fabric8-quickstarts>
+<https://github.com/fabric8-images>
+
+## Building and deploying
```

Comment:
> No, I mean put the intro at the top, followed by what this repo is about.

Its probably because its 1 am that I can't grok what you mean as the intro is already at the top - thus I've pushed this current version (with a few minor fixes) to master so you can do a quick PR to move the sections around as you think - then I can add/adjust any extra necessary text from there tomorrow ;)